cloudferrylib/os/compute/nova_compute.py

```diff
@@ -79,6 +80,12 @@
                                     for status in ALLOWED_VM_STATUSES}
 
 
+class FailAfterCreateInstance(Exception):
```

Comment:
I suggest use a `RuntimeError` instead of `Exception` as a parent, because `Exception` class is too general